src/main/java/pl/betoncraft/betonquest/config/Config.java

```diff
@@ -539,7 +539,7 @@ private void searchForPackages(final File file) {
                 for (final File subFile : content) {
                     searchForPackages(subFile);
                 }
-            } catch (final Exception e) {
+            } catch (final SecurityException e) {
```

Comment:
We shouldn't catch the `SecurityException` here as it only happens if the Java's security manager is used (it allows setting restrictions for system resources). This should never happen for a properly set up minecraft server.

Probably the best solution is to entirely remove the try-catch.

Also, the code may throw:
- `NullPointerException`s (`File.listFiles()` returns `null` on IO errors) and
- `IllegalStateException`s (`ConfigPackage` constructor creates `ConfigAccessor` objects whose constructor may throw those).